hlink/spark/session.py

```diff
@@ -29,7 +29,8 @@ def __init__(self, derby_dir, warehouse_dir, tmp_dir, python, db_name):
         self.python = python
 
     def spark_conf(self, executor_cores, executor_memory, driver_memory, cores):
-        spark_package_path = os.path.dirname(hlink.spark.__file__)
+        # Was getting an error where hlink.spark.__file__ return None and everything breaks
+        spark_package_path = os.path.dirname(hlink.spark.session.__file__)
```

Comment:
Hmm, this shouldn't be happening. Do you have an hlink/spark/__init__.py file in your repo?

Reply:
Yep, which is why this felt super weird. It's empty, thus `hlink.spark.__file__` returns None and crashes things when it tries to find the `dirname(None)`. This would return the same either way though, right?

Reply:
It should return a path even though the \_\_init\_\_.py file is empty. We want to avoid changing this path if possible.

How have you set up your environment? Have you called `pip install -e .[dev]`? It could be that there's something weird going on with your environment.